chromadb/embedding_functions/__init__.py

```diff
@@ -0,0 +1,138 @@
+from typing import Dict, Any, Type
```

Comment:
if these new EFs are backwards compatible can we delete the current EFs? would also make the diff easier to read by (hopefully) only showing changes.